project.clj

```diff
@@ -252,9 +212,11 @@
 
   :jvm-opts ["-Djruby.logger.class=com.puppetlabs.jruby_utils.jruby.Slf4jLogger"
                "-XX:+UseG1GC"
-               ~(str "-Xms" (heap-size "1G" "min"))
-               ~(str "-Xmx" (heap-size "2G" "max"))
-               "-XX:+IgnoreUnrecognizedVMOptions"]
+               ~(str "-Xms" (heap-size "1G"))
+               ~(str "-Xmx" (heap-size "2G"))
+               "-XX:+IgnoreUnrecognizedVMOptions"
+               "--add-modules=java.xml.bind"
+               "--add-modules=java.xml.ws"]
```

Comment:
Looks like this came with the merge up?

Reply:
Ah yeah, I guess those weren't part of those changes.